lib/open_api_spex/inspect/for_schema.ex

```diff
@@ -11,6 +11,12 @@ defimpl Inspect, for: OpenApiSpex.Schema do
       end)
       |> Map.new()
 
-    concat(["%OpenApiSpex.Schema", to_doc(map, opts)])
+    infos =
+      for %{field: field} = info <- OpenApiSpex.Schema.__info__(:struct),
+          Map.has_key?(map, field),
+          field not in [:__struct__, :__exception__],
```

Comment:
Can you explain why `field not in [:__struct__, :__exception__]` is necessary?

```elixir
iex(1)>  OpenApiSpex.Schema.__info__(:struct)
[
  %{field: :title, required: false},
  %{field: :multipleOf, required: false},
  %{field: :maximum, required: false},
  %{field: :exclusiveMaximum, required: false},
  %{field: :minimum, required: false},
  %{field: :exclusiveMinimum, required: false},
  %{field: :maxLength, required: false},
  %{field: :minLength, required: false},
  %{field: :pattern, required: false},
  %{field: :maxItems, required: false},
  %{field: :minItems, required: false},
  %{field: :uniqueItems, required: false},
  %{field: :maxProperties, required: false},
  %{field: :minProperties, required: false},
  %{field: :required, required: false},
  %{field: :enum, required: false},
  %{field: :type, required: false},
  %{field: :allOf, required: false},
  %{field: :oneOf, required: false},
  %{field: :anyOf, required: false},
  %{field: :not, required: false},
  %{field: :items, required: false},
  %{field: :properties, required: false},
  %{field: :additionalProperties, required: false},
  %{field: :description, required: false},
  %{field: :format, required: false},
  %{field: :default, required: false},
  %{field: :nullable, required: false},
  %{field: :discriminator, required: false},
  %{field: :readOnly, required: false},
  %{field: :writeOnly, required: false},
  %{field: :xml, required: false},
  %{field: :externalDocs, required: false},
  %{field: :example, required: false},
  %{field: :deprecated, required: false},
  %{field: :"x-struct", required: false},
  %{field: :"x-validate", required: false},
  %{field: :extensions, required: false}
]
```

I don't see how field can be either `__struct__` or `__exception__.